### PR TITLE
[5.x] Release prep for prereleases

### PR DESCRIPTION
**Pull request by frederikja163**
### Purpose of this PR
Small last release preps for otk 5.0-pre.1

### Testing status
Currently testing this on the learnopentk repo with an example triangle. Will keep adding fixes here as i go.

### CODE REVIEW

File: src/OpenTK.Graphics/OpenGL/Compatibility/GL.Manual.cs [hunk 1]
```diff
@@ -5,6 +5,13 @@ namespace OpenTK.Graphics.OpenGL.Compatibility
 {
     public static unsafe partial class GL
     {
+        // FIXME: Remove this when it's fixed
+        // This is here because there in the gl.xml
+        // one of the paramteres for "glSampleMaskIndexedNV"
+        // is marked with a group named this, but this group is never referenced
+        // anywhere else in the file.
+        public enum SampleMaskNV
```
Comment (PJB3005) [inline src/OpenTK.Graphics/OpenGL/Compatibility/GL.Manual.cs:13]: Doesn't the bindings generator have the ability to manually fit overrides like these in? Or does that not work with enums.

Comment (frederikja163) [reply src/OpenTK.Graphics/OpenGL/Compatibility/GL.Manual.cs:13]: Read the comments, this one is broken in the .xml files. We will fix it but this fix will do for the first pre-release

Comment (frederikja163) [reply src/OpenTK.Graphics/OpenGL/Compatibility/GL.Manual.cs:13]: The only change i did here is making the API's mirror each other, as previously this was only implemented for `Graphics.OpenGL`

File: src/OpenTK.Graphics/OpenGL/Compatibility/GL.Manual.cs [hunk 2]
```diff
@@ -5,6 +5,13 @@ namespace OpenTK.Graphics.OpenGL.Compatibility
...
+        { }
         public static void ShaderSource(uint shader, string str)
```
Comment (NogginBops) [inline src/OpenTK.Graphics/OpenGL/Compatibility/GL.Manual.cs:15]: One more line here plz

**Pull request merged**